docs/guides/universal-receiver-delegate/create-custom-urd-2.md

```diff
@@ -0,0 +1,294 @@
+---
+sidebar_label: 'Create a custom Forwarder URD (2/2)'
+sidebar_position: 2
+---
+
+# Deploy and test our custom forwarder URD
+
+In the [first part](./create-custom-urd-1.md), we setup our Universal Profile and created the custom URD contract. We will now deploy and test it!
+
+## Create a MockContract to generate the UniversalProfile type
+
+In order to deploy this Custom URD contract, we will interact with a UniversalProfile. We can enhance the developer experience by generating the types for a `UniversalProfile` contract by creating a contract that will import it. Create a `MockContract.sol` file in the `contracts/` file with the following content:
+
+```solidity title="contracts/MockContract.sol"
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+import {UniversalProfile} from '@lukso/lsp-smart-contracts/contracts/UniversalProfile.sol';
+```
+
+## Deploy our custom URD
+
+As we have 2 different methods with specific requirements, we will provide both set of code. Comment the code that doesn't match your choice.
+
+Now we can create the deploy scripts that will connect all the pieces together. This script will:
+
+- Connect to our UP
+- Deploy the custom URD as our UP
```

Comment:
What is meant here by _"as our UP"_? You are using the Universal Profile to deploy the custom URD contract? Might be good to clarify, as I am not sure I understood.

Reply:
changed explanation with a better one